src/view/trip-info.js

```diff
@@ -0,0 +1,53 @@
+import { createElement } from '../render.js';
+
+function createHeaderTemplate() {
```

Comment:
Любой шаблон -- это просто строка/текст. Т.е. функция возвращает строку, если мы хотим сделать вложенные функции без вынесения вьюшек, мы можем какие-то части кода вынести так же в функции, возвращающие строки с ними и тут их вклеить, как в любую обычную строку:
```suggestion
function createInfoViewTemplate() {
  return (`<section class="trip-main__trip-info  trip-info">
  ${createTitleTemplate()}
  ${createTotalPriceTemplate()}
</section>`);

function createTitleTemplate() {
  return (`<div class="trip-info__main">
  <h1 class="trip-info__title">Amsterdam &mdash; Chamonix &mdash; Geneva</h1>

  <p class="trip-info__dates">Mar 18&nbsp;&mdash;&nbsp;20</p>
</div>`);
}

function createTotalPriceTemplate() {
  return (`<p class="trip-info__cost">
  Total: &euro;&nbsp;<span class="trip-info__cost-value">1230</span>
</p>`);
}
}
```